controllers/pitch.js

```diff
@@ -1,39 +1,40 @@
 var Pitch = require("../models/Pitch");
 
 module.exports = {
-  insert: function(req, res) {
+  insert: function (req, res) {
     Pitch.create(req.body)
-      .then(function(doc) {
+      .then(function (doc) {
         res.json(doc);
       })
-      .catch(function(err) {
+      .catch(function (err) {
         res.json(err);
       });
   },
-  getAllProjects: function(req, res) {
+  getAllProjects: function (req, res) {
     Pitch.find(req.query)
-      .then(function(doc) {
+      .sort({ date: -1 })
```

Comment:
This line sorts the results by date in descending order.

Reply:
That would be a much more user-friendly interface.